src/reparentBundleTarget.ts

```diff
@@ -0,0 +1,65 @@
+import { isLocalRef } from './isLocalRef';
+import { isPlainObject } from './isPlainObject';
+import { pointerToPath } from './pointerToPath';
+
+function isObject(maybeObj: unknown): maybeObj is Record<string, unknown> | unknown[] {
+  return isPlainObject(maybeObj) || Array.isArray(maybeObj);
+}
+
+export function reparentBundleTarget(document: Record<string, unknown>, from: string, to: string): void {
```

Comment:
Could you add a comment here describing what this function does and an example of how we use it? Because yea, difficult to give this one a name that makes sense 😂.